src/pages/resultsView/styles.scss

```diff
@@ -1,29 +1,55 @@
 #oncoprint {
-  border:none !important;
-  padding:0 !important;
-
+  border: none !important;
+  padding: 0 !important;
 
   #oncoprint-inner {
     .oncoprintjs__scroll_div {
-      overflow:auto !important;
+      overflow: auto !important;
+    }
+
+    .noselect {
```

Comment:
@adamabeshouse if it's not too hard, would be nice to have classes on the track controls button and the dropdown